rcsbsearch/search.py

```diff
@@ -392,8 +392,13 @@ class Attr:
     Pre-instantiated attributes are available from the
     :py:data:`rcsbsearch.rcsb_attributes` object. These are generally easier to use
     than constructing Attr objects by hand. A complete list of valid attributes is
-    available in the `schema <http://search.rcsb.org/rcsbsearch/v1/metadata/schema>`_.
+    available in the `schema <http://search.rcsb.org/rcsbsearch/v2/metadata/schema>`_.
 
+    * The `range` dictionary requires the following keys:
+      * "from" -> int
+      * "to" -> int
+      * "include_lower" -> bool
+      * "include_upper" -> bool
```

Comment:
It's unclear to me if this is the best change of API that we want in the library, but it is a direct change to the [new format specified in v2](https://search.rcsb.org/migration/migration-guide-v1-to-v2.html). Open to suggestions!